class_person.rb

```diff
@@ -2,12 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] Great job implementing the `Person` class and setting up **Setter** and **Getter** for the instance variables `@name` and `@age` :clap: However, it's important that we set up both **Setter** and **Getter** for the `@rentals` variable as well. That will help to manage a person's rental outside the `Person` class, like adding a rental to a person in the `Rental` class. So Kindly set up both **Getter** and **Setter** for the `@rentals` variable as you did for the `@name` and `@age`

Reply:
- Set up both Getter and Setter for the `@rentals` variable